apps/src/applab/designMode.js

```diff
@@ -925,24 +925,27 @@ designMode.createScreen = function () {
 /**
  * Changes the active screen by triggering a 'CHANGE_SCREEN' action on the
  * Redux store.  This change propagates across the app, updates the state of
- * React-rendered components, and eventually calls onScreenChange, below.
+ * React-rendered components, and eventually calls renderScreens, below.
  * @param {!string} screenId
  */
 designMode.changeScreen = function (screenId) {
   studioApp.reduxStore.dispatch(screens.changeScreen(screenId));
 };
 
 /**
- * Responds to changing the active screen by toggling all screens to be
- * non-visible, unless they match the provided screenId, and opens the element
+ * Given the currentScreenId in our redux store, toggles all screens to be
+ * non-visible, unless they match the provided screenId, with the element
  * property editor for the new screen.
  *
  * This method is called in response to a change in the application state.  If
  * you want to change the current screen, call designMode.changeScreen instead.
  *
  * @param {!string} screenId
  */
-function onScreenChange(screenId) {
+function renderScreens(screenId) {
```

Comment:
I might attempt a larger refactor of onInterfaceModeChange in a separate PR as I suspect that at least some of what it's doing can be moved into components at this point. I'm realizing that designWorkspace in particular is somewhat strange for a couple reasons:
1. We create a div with this id in `CodeWorkspaceContainer` which is used by every app, even though I'm pretty sure App Lab is the only one that wants a designWorkspace.
2. We create it as a protected div, but then still render a react component there using `designMode.renderDesignMode`.